src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -915,7 +915,7 @@ public ResultSet getColumns(String c, String s, String tblNamePattern, String co
         checkOpen();
 
         StringBuilder sql = new StringBuilder(700);
-        sql.append("select null as TABLE_CAT, null as TABLE_SCHEM, tblname as TABLE_NAME, ")
+        sql.append("select null as TABLE_CAT, ").append(quote(s)).append(" as TABLE_SCHEM, tblname as TABLE_NAME, ")
```

Comment:
if `s` is null, then will it quote the `NULL` string thus returning `"NULL"` instead of the SQL `null` ?

Reply:
```
    protected static String quote(String tableName) {
        if (tableName == null) {
            return "null";
        } else {
            return String.format("'%s'", tableName);
        }
    }
```
It will be null for SQL requests. And also, it can be seen by unit tests that it will result in a `null` value.